public/app/features/provisioning/ExportToRepository.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect } from 'react';
```

Comment:
I would have imagined a much simpler frontend change for this first iteration. 

I would: 
- Add a button `Export` in the Repository Page above the sections. This button will be disabled if there is an ongoing export job. You could also create the alert in this chunk. 
- In the existing `Recent Events`, users can see the status already of the job. So it will be a noop.

Reply:
I'm not totally sure -- I am not sure we want an "export" shown anywhere except the initial setup wizard.  The goal here was to have *something* that can watch the single export job that would eventually end up in the wizard.